TOC.md

```diff
@@ -98,7 +98,7 @@
     - [Hybrid Topology](/hybrid-deployment-topology.md)
   - Install and Start
     - [Use TiUP](/production-deployment-using-tiup.md)
-    - [Deploy in Kubernetes](/tidb-in-kubernetes.md)
+    - [Deploy on Kubernetes](/tidb-in-kubernetes.md)
```

Comment:
[`configure-placement-rules.md` L45](https://github.com/ran-huang/docs/blob/on-k8s/configure-placement-rules.md?plain=1#L45)

```
`LabelConstraint` is similar to the function on Kubernetes that filters labels based on these four primitives: `in`, `notIn`, `exists`, and `notExists`. The meanings of these four primitives are as follows:
```

Reply:
I think we can keep "in" for this case.
TiDB Operator is deployed on Kubernetes.
A function (feature) is used in Kubernetes, not on Kubernetes.